src/Entity/Developer.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\DevRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=DeveloperRepository::class)
+ */
+class Developer
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\OneToOne(targetEntity="User")
+     * @ORM\JoinColumn(name="user_id", referencedColumnName="id")
+     */
+    private $user_id;
```

Comment:
Please change the properties to macht camelCase instead of snake_case, this is a convention we should use for all variables within php. Also if you use the php bin/console make:entity command it will output the properties as camelCase :)